openmdao/docs/features/core_features/grouping_components/configure_method.rst

```diff
@@ -26,19 +26,5 @@ system.
 Uses of setup vs. configure
 ---------------------------
 
-**setup**
-
- - Add subsystems
- - Issue connections
- - Assign linear and nonlinear solvers at group level
- - Change solver settings in group
- - Assign Jacobians at group level
- - Set system execution order
-
-**configure**
-
- - Assign linear and nonlinear solvers to subsystems
- - Change solver settings in subsystems
- - Assign Jacobians to subsystems
- - Set execution order in subsystems
+To understand when to use setup and when to use configure, see the :ref:`Theory Manual entry on how the setup stack.<theory_setup_stack>`.
```

Comment:
"see the Theory Manual entry on how the setup stack." seems like a sentence fragment, assume you want to add a word like "works" to the end?